### PR TITLE
Issue#55 create database models

### PR DESCRIPTION
**Pull request by aasimdag**

### CODE REVIEW

File: artifact_backend/.env
Comment (sametaln) [inline artifact_backend/.env:1]: We should add this file to .gitignore.

File: artifact_backend/.idea/.gitignore
Comment (sametaln) [inline artifact_backend/.idea/.gitignore:1]: add .idea to .gitignore

**Comment by sametaln** [comment]
Updated the dockerfile and docker-compose files to be compatible with the planned codebase structure.

**Comment by sametaln** [comment]
Changed the base branch to backend because the parent of the branch `issue#55-create-database-models` was `backend`.

**Comment by oguzhekim** [comment]
I edited the models to reflect the relations between them better. Recreated migration files according to new models.

**Pull request merged**